pickle2graphite.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, metricsDict, base):
                 (year, month, day) = devAttrs["Date"].split("-")
                 (hour, minute, second) = devAttrs["Time"].split(":")
             except KeyError:
-                raise ValueError("Date or Time is missing or incorrrectly formatted for this set of metrics")
-                raise
```

Comment:
I agree that, unless you are running in debug mode, you want to just drop bad data and keep running.  It is worthwhile to capture the raw data using something like tee so that you can go back and re-process it if there was an error and something was lost.

`tcpdump | tee capture.pcap | seextract | semonitor | pickle2graphite
`

Note that it is advisable to use seextract.py or some other utility to extract the TCP streams from the capture.  Reading a pcap file directly into semonitor.py will sort of work, but you will get spurious errors and may lose some data.